src/main/java/com/github/tomakehurst/wiremock/matching/ContentPatternDeserialiser.java

```diff
@@ -16,20 +16,18 @@
 package com.github.tomakehurst.wiremock.matching;
 
 import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
-import com.fasterxml.jackson.databind.JsonMappingException;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.ImmutableList;
 import java.io.IOException;
-import java.util.Map;
+import java.util.*;
+import java.util.stream.StreamSupport;
 
 public class ContentPatternDeserialiser extends JsonDeserializer<ContentPattern<?>> {
 
   @Override
   public ContentPattern<?> deserialize(JsonParser parser, DeserializationContext context)
-      throws IOException, JsonProcessingException {
```

Comment:
The class is not final, so it is better to not remove it to avoid formally a compatibility breaking change

Reply:
I will make it in next update)